lib/mail_room/arbitration/redis.rb

```diff
@@ -44,7 +45,12 @@ def deliver?(uid)
 
       def redis
         @redis ||= begin
-          redis = ::Redis.new(url: options.redis_url)
+          sentinels = options.sentinels
+          if sentinels
+            redis = ::Redis.new(url: options.redis_url, sentinels: sentinels)
+          else
+            redis = ::Redis.new(url: options.redis_url)
```

Comment:
I don't really like repeating `::Redis.new(url: options.redis_url)`. What do you think about:

```ruby
redis_options = {}
redis_options[:url] = options.redis_url
redis_options[:sentinels] = options.sentinels if options.sentinels
::Redis.new(redis_options)
```